### PR TITLE
[Network] Post, Delete API 연결 (#41)

### LINKED ISSUE #41: [Network] API 연결 (post, delete)
## 🌟 Issue

- API 연결

## 📝 To-do

- [x] post
- [x] delete

## 🕓 예상 작업 기한

- 1일

### PR DESCRIPTION
**Pull request by mcrkgus**
# 🔥 *Pull requests*

## ⛳️ **작업한 브랜치**
- #41 

## 👷 **작업한 내용**

- bookmark post delete 

## 💫 관련 이슈
- Resolved: #41

### CODE REVIEW

File: LINKAREER-iOS/LINKAREER-iOS/Network/NewbieNetwork/NewbieTargetType.swift [hunk 1]
```diff
@@ -10,34 +10,58 @@ import Moya
...
+    case postBookMark(officialId: Int)
+    case deleteBookMark(officialId: Int)
 }
 
 extension NewbieTargetType: BaseTargetType {
     var utilPath: String {
-        // 게시물 API의 기본 경로
-        return "api/v1/officiallist"
+        switch self {
+        case .getPostList(let category):
```
Comment (Ohjackson) [inline LINKAREER-iOS/LINKAREER-iOS/Network/NewbieNetwork/NewbieTargetType.swift:20]: goood~!

File: LINKAREER-iOS/LINKAREER-iOS/Presentation/NewBieIntern/Controller/NewbieInternViewController.swift [hunk 2]
```diff
@@ -339,8 +339,14 @@ extension NewbieInternViewController: UICollectionViewDataSource, UICollectionVi
 }
 
 
+extension NewbieInternViewController: BookMarkDelegate {
+    func didTapActionButton(in cell: CompanyDayCardCell) {
+        cell.actionButton.isSelected ? deleteBookmarkAPI() : postBookmarkAPI()
```
Comment (Ohjackson) [inline LINKAREER-iOS/LINKAREER-iOS/Presentation/NewBieIntern/Controller/NewbieInternViewController.swift:344]: gooood job ~!

File: LINKAREER-iOS/LINKAREER-iOS/Presentation/Home/ViewController/HomeViewController.swift [hunk 3]
```diff
@@ -25,7 +25,7 @@ class HomeViewController: UIViewController {
     private var categorySelector: [CategorySelector] = CategorySelector.dummyData
     private var homeBanners: [HomeBanner] = HomeBanner.dummyData
 
-    private var interestBoard: [Board] = Board.dummyData
+    private var interestBoard: [Board] = []
```
Comment (Ohjackson) [inline LINKAREER-iOS/LINKAREER-iOS/Presentation/Home/ViewController/HomeViewController.swift:28]: Thank you for your kindness!

**Review by Ohjackson — APPROVED** [review]
수고하셨어요~~~! LGTM~~~!

**Review by kms0233 — APPROVED** [review]
냠! 고생했다아아ㅏ ㅠㅠㅠ🤍

**Pull request merged**